Dockerfile

```diff
@@ -420,6 +420,17 @@ RUN mkdir -p $BAZEL_HOME/bin \
 FROM scratch as bazel
 COPY --from=bazelbuild /opt/bazel /opt/bazel
 
+#------------------------------------------------------------------------
+# BITBAKE
+FROM base as bitbakebuild
+
+ARG BITBAKE_VERSION=2.8.0
+
+RUN git clone --branch $BITBAKE_VERSION --depth 1 https://git.openembedded.org/bitbake /opt/bitbake
```

Comment:
Why did you choose to clone the _branch_, not the _tag_? 

Note: There is also an archive URL available https://git.openembedded.org/bitbake/snapshot/bitbake-2.8.0.tar.gz.

Reply:
There is no `--tag` for `git clone`, but [`--branch` "can also take tags"](https://git-scm.com/docs/git-clone#Documentation/git-clone.txt-code--branchcodeemltnamegtem).